apps/mobile/src/libs/components/wheel-loader/wheel-loader.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { Wheel } from "~/libs/components/components";
+import { AnimationName, GradientColor } from "~/libs/enums/enums";
+
+type Properties = {
+	animationDuration: number;
+	size: number;
+};
+
+const gradientSectorsData = [
```

Comment:
Move to a separate file

Reply:
@fshabanov  Not sure that we need to move mock data to a separate file.